pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: layout.pug
+navigationTitle: Hosting on Gitea
+title: Hosting on Gitea
+menuWeight: 10
+excerpt: How to host git repositories on Gitea for airgapped deployments
+---
+
+DKP Supports using an external catalog git repository to install applications. In cases where an external repository is not accessible from within the cluster, it is possible to use the Gitea server that comes out-of-box with a kommander deployment to install applications.
+
+
+1. Set the `VERSION` environment variable to the version of Kommander you want to install, for example:
+
+    ```bash
+    export VERSION=v2.1.1
+    ```
+
+1. Set the `TARGET_NAMESPACE` to the workspace (or project) namespace in which the catalog repository needs to be created.
+
+    ```bash
+    export TARGET_NAMESPACE=<WORKSPACE_OR_PROJECT_NAMESPACE>
+    ```
+
+1. Create environment variables that contain the Gitea credentials:
+
+    ```bash
+    GITEA_USERNAME=$(kubectl get secrets -nkommander-flux kommander-git-credentials -oyaml -o go-template="{{.data.username | base64decode }}")
+    GITEA_PASSWORD=$(kubectl get secrets -nkommander-flux kommander-git-credentials -oyaml -o go-template="{{.data.password | base64decode }}")
+    ```
+
+1. Using the above username and password, follow the steps below to login to the gitea UI and create a new repository in the `kommander` organization.   
+    
+    ```bash
+    GITEA_URL=$(kubectl -n kommander get ingress gitea -o go-template='https://{{ (index .status.loadBalancer.ingress 0).hostname }}:443/dkp/kommander/git')
+    ```
+
+1. Login to the Gitea dashboard at `GITEA_URL` with the `GITEA_USERNAME` and `GITEA_PASSWORD` credentials. Create a new repository named `dkp-catalog-applications` (or any other name you prefer) under `kommander` organization.
+
+1. Clone the newly created repository on your local machine:
+
+    ```bash
+    git clone -c http.sslVerify=false https://$GITEA_USERNAME:$GITEA_PASSWORD@$GITEA_URL/kommander/dkp-catalog-applications
```

Comment:
Our security-aware customers will not like this. Please add instructions using the CA file from the cluster

Reply:
A kind of more secure way (at least one that doesn't completely bypass certificate checks) would be
```
k -n kommander get secret kommander-traefik-certificate -o go-template='{{index .data "ca.crt"|base64decode}}' > /tmp/ca.crt && git clone -c http.sslCAInfo=/tmp/ca.crt https://$(k -n kommander get secret admin-git-credentials -o go-template='{{.data.username|base64decode}}:{{.data.password|base64decode}}')@$((k -n kommander get cm konvoyconfig-kubeaddons -o go-template='{{if ne .data.clusterHostname ""}}{{.data.clusterHostname}}{{"\n"}}{{end}}' && k -n kommander get ingress gitea -o jsonpath="{.status.loadBalancer.ingress[0]['ip','hostname']}") | head -1)/dkp/kommander/git/kommander/kommander && rm /tmp/ca.crt
```

Reply:
i replaced `&&` with `;` so that this doesnt fail when if `cm` is not present (not that it ever would be, just in case). 

Updated the instructions to create a new gitea user instead of copying kommander user credentials.